src/creature.ts

```diff
@@ -96,7 +96,7 @@ export class Creature {
 	 * name :			String :	Creature name
 	 * id :			Integer :	Creature Id incrementing for each creature starting to 1
 	 * size :			Integer :	Creature size in hexes (1,2 or 3)
-	 * type :			Integer :	Type of the creature stocked in the database
+	 * type :			String :	Type of the creature stocked in the database. Made up of `creature.realm` + `creature.level`. ex: for Dark Priest "--"
```

Comment:
This should say "exception", not "ex", because that's usually taken for "example" in most parts of the world. The way British people use it is pretty opposite to that, rather a counterexample; they also drive on the other side of the road and use pretty bad measuring units 🐻